clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -246,7 +246,10 @@ public LeaderFollowerStoreIngestionTask(
             .build();
     this.veniceWriter = Lazy.of(() -> veniceWriterFactory.createVeniceWriter(writerOptions));
     this.kafkaClusterIdToUrlMap = serverConfig.getKafkaClusterIdToUrlMap();
-    this.kafkaDataIntegrityValidatorForLeaders = new KafkaDataIntegrityValidator(kafkaVersionTopic);
+    this.kafkaDataIntegrityValidatorForLeaders = new KafkaDataIntegrityValidator(
+        kafkaVersionTopic,
+        KafkaDataIntegrityValidator.DISABLED,
```

Comment:
Why is this hardwired to DISABLED?

Reply:
So... this 2nd param is used in a code path that's not used by anything in open source... but the proprietary ETL populates that param. It is kind of messy, admittedly, but I fear refactoring this is a bit complex for now.

Reply:
Ah I see.  Ok.  Maybe throw a TODO at the interface in the class so no one deletes it later while "cleaning up"